docs/api-docs/apps/guide/apps-03-developing.md

```diff
@@ -52,22 +53,8 @@ ngrok http 3000                  # start ngrok
 
 For step-by-step instructions, see [How to Test App Authentication Locally with ngrok](https://medium.com/bigcommerce-developer-blog/how-to-test-app-authentication-locally-with-ngrok-149150bfe4cf) on our [developer blog](https://medium.com/bigcommerce-developer-blog).
 
-<div class="HubBlock--callout">
-<div class="CalloutBlock--info">
-<div class="HubBlock-content">
-
-<!-- theme: info -->
-
-### Note
-> * If you'd rather skip setting up a development environment and experiment with app development in the browser, fork our [Express Hello World App in CodeSandbox](https://codesandbox.io/s/express-hello-world-app-fq5t1?file=/app.js). For step-by-step instructions, see the [Building Apps Quick Start Tutorial](https://developer.bigcommerce.com/api-docs/apps/quick-start).

```

Comment:
@markcmurphy @slsriehl yes, the codesandbox repo has been deprecated - this reference should be updated to point to the new sample app. 

Please note: the Dev Docs Updates requests that the references be replaced, not removed. There is still value in retaining the link the the Quick Start Tutorial. Additionally, since the new sample app has a one-click deploy to Heroku, we should simply update the wording here. e.g. "If you’d rather skip setting up a development environment and experiment with app development in the browser, deploy the Sample App to Heroku." The Heroku deploy button can be found here: https://github.com/bigcommerce/sample-app-nodejs#readme

Reply:
I'd be very OK adding a note to the old codesandbox app. The owner of the codesandbox is listed as BC Developer Documentation - we'll need to figure out who has access to modify the template.

Reply:
@bc-chaz  

### generally:
In the larger context of these changes to this page, this callout duplicates and obscures.  It seems to me that its original intent was to say "impatient people, it's so easy to get started codesandbox, go!"  One of the costs of moving to React everything is the heat death of the Hello World.  I attempted to adjust expectations by retitling `## Beginning with hello world` as `## Beginning quickly`, and linking the `quick-start` tutorial & clonable repos from there. (lines `24`-`32`).

### specifically:
`apps-03-developing.md` is intended to be sample app agnostic.  IMO it's not appropriate to link here a particular Heroku deploy button or content specific to the `/sample-app-nextjs/` tutorial (repo `sample-app-nodejs`).

The `quick-start` tutorial referenced here is still linked at:
- `apps-01-intro...md` lines `15` & `79`
- `apps-02-...md` line `63`
- (this document)`apps-03-developing.md`  lines `25` and `83`.

The heroku deploy button & instructions are available in the `introduction` to the `/sample-app-nextjs/` tutorial, as well as at the repo readme you mentioned on line `27`.

Reply:
> I'd be very OK adding a note to the old codesandbox app. The owner of the codesandbox is listed as BC Developer Documentation - we'll need to figure out who has access to modify the template.

@markcmurphy I have a couple ideas -- maybe SSO from our delegated address, or a couple likely previous users.  follow up is in dd slack.

Reply:
I get where you're coming from with wanting this page to be more agnostic. I'd counter, however, that it can be a bit overwhelming with so many options. Ideally, we should be able to highlight a quick path/ specific app - given that the sample app is our latest application, it would make logical sense to promote it. I believe Shopify does something similar.

Another consideration is that many, if not all, of the other sample apps are now a bit dated. While it is on our radar to translate the React Sample in multiple languages, the React Sample App is currently the only one that captures all of our best practices (including the BC SDK, BigDesign, etc). For that reason, I'd recommend promoting the new sample app where we can. 

Lastly, this page is labeled, "Beginning App Development." Developers are coming here for guidance on where to get started - some of whom may wish for a "quick start." Linking to the quick start on other pages doesn't preclude us from doing so here. Given that this is literally the page for where to get started, we should devs that resource.

Reply:
revised with [commit 3b89b94](https://github.com/bigcommerce/dev-docs/pull/1337/commits/3b89b9485d9698d7ce5a11b4fb54966ce596e3ba) & @bc-chaz approved in slack.

Reply:
re note in deprecated codesandbox app -- ticket created, [DEVDOCS-3298](https://jira.bigcommerce.com/browse/DEVDOCS-3298).

Reply:
2021-05-06 -- CodeSandbox account recovered and [old sample app marked deprecated](https://codesandbox.io/s/fq5t1) and linked to new sample app repo.

cc @bc-chaz @markcmurphy